.circleci/config.yml.in

```diff
@@ -32,12 +32,8 @@ commands:
       - run:
           name: adding UPLOAD_CHANNEL to BASH_ENV
           command: |
-            our_upload_channel=nightly
-            # On tags upload to test instead
-            if [[ -n "${CIRCLE_TAG}" ]] || [[ ${CIRCLE_BRANCH} =~ release/* ]]; then
-              our_upload_channel=test
-            fi
-            echo "export UPLOAD_CHANNEL=${our_upload_channel}" >> ${BASH_ENV}
+            # Hardcoded for release branch
+            echo "export UPLOAD_CHANNEL=test" >> ${BASH_ENV}
```

Comment:
Since we  on the release branch, why is this change required ? It should automatically use test, correct ?

Reply:
According to @mthrok : Even though UPLOAD_CHANNEL is set to test if the branch name is release/*, when you are making a PR against the release branch, the value of CIRCLE_BRANCH is something like pr/XXX, and the CI jobs of PRs against release branches use nightly builds.

Reply:
That's correct. We need this for invoking signal for CI like. CicleCI's spec could have changed but it's safe to hard code this.

https://github.com/pytorch/audio/pull/1690

Reply:
Got it! Thanks for explanation.